### ISSUE TITLE
The boost module and bubble series

### ISSUE THREAD

**Issue by PaulDalek** [highcharts/highcharts#7225]
#### Expected behaviour
Colors of bubbles are preserved when boostThreshold is set to 0, when using boost module.

#### Actual behaviour
The lower opacity value of colors is set for each bubble.

The latest working version is 5.0.7.

#### Live demos
http://jsfiddle.net/c9Lpbx3w/ - opacity is set (latest version, v6.0.0)
http://jsfiddle.net/uea01g0m/ - colors are preserved (v5.0.7)

**Comment by sebastianbochan** [comment]
_Internal note_
Colors are changed since 5.0.8.

**Comment by KacperMadej** [comment]
It looks like the boost module is here to show how the chart should work, because in marker options for series there's fillOpacity - if it will be set then all works fine. By default it's `0.5`.

Demo: http://jsfiddle.net/BlackLabel/c9Lpbx3w/1/

Colors set by name are not correctly affected by the setting, but boost is translating all colors to it's hex codes, so this might be the reason.

Green set by hex code is affected: http://jsfiddle.net/BlackLabel/c9Lpbx3w/2/

**Comment by cvasseng** [comment]
This looks to be an issue with the color system in Highcharts - none of the fiddles posted above are actually boosting. You can tell by the point animation present on all of them, and/or by inspecting the bubbles (they're all `<path>` elements, so it's in SVG-mode). 

Setting `boostThreshold` to `0` disables boost for that series. It has to be `1` to force boost.

There is another issue though, I can't get boost to work properly with the data as it's defined in the fiddles (it won't take the sizing into consideration).

**Comment by TorsteinHonsi** [comment]
Bubble series have a `fillOpacity` of 0.5 by default. It was not documented in the API, but I have fixed that (upcoming).

In Highcharts, named colors are only [partially supported](https://www.highcharts.com/docs/chart-design-and-style/colors), meaning opacity is not applied to them because Highcharts doesn't understand what `red` is. That is why, when the Boost module is _not_ included, the colors are solid. 

Now when we include the Boost module, we also include a list of aliases for color names that we need for the Boost module to work. So now `red` is internally translated to `#ff0000`, Highcharts is able to parse it, apply fill opacity, and subsequently we have semi-transparent colors.

In summary, use hex colors instead of named colors for a reliable result.
